rocks-lib/src/operations/install.rs

```diff
@@ -26,11 +26,16 @@ use super::SearchAndDownloadError;
 #[derive(Error, Debug)]
 #[error(transparent)]
 pub enum InstallError {
+    #[error(transparent)]
     SearchAndDownloadError(#[from] SearchAndDownloadError),
+    #[error(transparent)]
     LuaVersionError(#[from] LuaVersionError),
+    #[error(transparent)]
     LuaVersionUnset(#[from] LuaVersionUnset),
+    #[error(transparent)]
```

Comment:
nitpick: because of the top-level `#[error(transparent)]` these extra statements here are redundant

Reply:
Removed the top-level one, because it doesn't apply to all constructors anymore.